src/index.ts

```diff
@@ -265,6 +265,17 @@ export const envVarsConfig = {
 	DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES,
 };
 
+// Needs to be `import *` so that the import path is preserved in the generated .d.ts
+import * as numericBigInteger from './translations/v6/numeric-big-integer-hack.js';
+/**
+ * This automatically augments the declarations of the consumer projects.
+ */
+export const augmentedTypes = {
+	'@balena/sbvr-types': {
+		'Numeric Big Integer': numericBigInteger,
+	},
+};
```

Comment:
You should be able to do this with just:
```suggestion
import './translations/v6/numeric-big-integer-hack.js';
```

Reply:
Loved it!